src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -262,6 +281,70 @@ public void subscribeAndCleanupResource_googleBigQueryTable() throws Exception {
     assertTrue(bigQueryCow.delete(datasetName));
   }
 
+  @Test
+  public void subscribeAndCleanupResource_googleProject() throws Exception {
+    String projectId = randomProjectId();
+
+    Operation operation =
+        resourceManagerCow
+            .projects()
+            .create(new Project().setProjectId(projectId).setParent(parentResourceId))
+            .execute();
+    OperationCow<Operation> operationCow = resourceManagerCow.operations().operationCow(operation);
+    operationCow =
+        OperationUtils.pollUntilComplete(
+            operationCow, Duration.ofSeconds(5), Duration.ofSeconds(30));
+    assertTrue(operationCow.getOperation().getDone());
+    assertNull(operationCow.getOperation().getError());
+
+    Project project = resourceManagerCow.projects().get(projectId).execute();
+    assertEquals(projectId, project.getProjectId());
+    assertEquals("ACTIVE", project.getLifecycleState());
```

Comment:
I don't think we need these asserts. In the CRL test, we're verifying that the Google APIs work how we expect. Here, I think it's fine to assume that we've created the project after a successful creation operation.